### PR TITLE
Wallet Onboarding - Full coverage.

### PR DESCRIPTION
**Pull request by anitha-deriv**
https://app.clickup.com/t/20696747/WALL-3039 - User Onboarding on Desktop for  Wallets having CFDs and Trading Accounts

### CODE REVIEW

File: cypress/e2e/wip/Wallets/cashier/onboarding.cy.js [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+  
+  //Wallet Onboarding -  tour around Options and Multiplier section
+  cy.get('.wallets-options-and-multipliers-listing__header').should('exist')
+    .then(($list) => {
+      const derivAppDesc = $list.children().eq(1).text()
+      const expderivAppDesc = 'Get a Deriv Apps trading account to trade options and multipliers on these apps.'
+      if (derivAppDesc.includes(expderivAppDesc)) {
+        cy.log(`Deriv App Trading Account is not created: ${expderivAppDesc}`)
+        cy.contains('Deriv Apps trading account')
+        .should('exist')
```
Comment (markodonnell-deriv) [inline cypress/e2e/wip/Wallets/cashier/onboarding.cy.js:35]: Alignment, needs one more tab. Same with lines 37 & 40

File: cypress/e2e/wip/Wallets/cashier/onboarding.cy.js [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+function onboardingfiatwallet() {
+    cy.contains("Wallet", { timeout: 10000 }).should("exist")
+    // Onboarding - Account having Fiat wallets (without CFD and Trading account).
+    cy.findByTestId('dt_traders_hub_onboarding_icon').click()
+    cy.get('.react-joyride__spotlight').should('exist')
+    cy.get('[class*="wallets-accordion wallets-accordion"]').first().should('be.visible')
+      .children()
+      .eq(1)
+      .get('.wallets-accordion__content.wallets-accordion__content--visible') //To verify if the USD wallet content is expanded instead of other wallets.
+      .should('exist')
```
Comment (markodonnell-deriv) [inline cypress/e2e/wip/Wallets/cashier/onboarding.cy.js:84]: Do we need an assert for exist if we have an assert for visible? Also does the test need to be so big. After the end() could it not be a separate assert?

File: cypress/e2e/wip/Wallets/cashier/onboarding.cy.js [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+      .should('be.visible')
+      .end()  
+      .get('.wallets-gradient--USD-desktop-card-light')
+      .should('exist')
+      cy.contains("This is your Wallet. These are the functions that you can perform within this Wallet and you can conveniently view your total balance here.")
```
Comment (markodonnell-deriv) [inline cypress/e2e/wip/Wallets/cashier/onboarding.cy.js:89]: Slight alignment issue.

**Review by markodonnell-deriv — CHANGES_REQUESTED** [review]

File: cypress/e2e/wip/Wallets/cashier/onboarding.cy.js (hunk 2 above)
Comment (anitha-deriv) [reply cypress/e2e/wip/Wallets/cashier/onboarding.cy.js]: Removed the assert for exist and corrected the alignments.

File: cypress/e2e/wip/Wallets/cashier/onboarding.cy.js (hunk 3 above)
Comment (anitha-deriv) [reply cypress/e2e/wip/Wallets/cashier/onboarding.cy.js]: Corrected the alignments

File: cypress/e2e/wip/Wallets/cashier/onboarding.cy.js (hunk 1 above)
Comment (anitha-deriv) [reply cypress/e2e/wip/Wallets/cashier/onboarding.cy.js]: Removed the assert for exist and corrected the alignments.

File: cypress/e2e/wip/Wallets/cashier/onboarding.cy.js (hunk 3 above)
Comment (anitha-deriv) [reply cypress/e2e/wip/Wallets/cashier/onboarding.cy.js]: Screenshot 2023-12-11 at 1 21 45 PM
Tests passed locally - after updating the code.

**Review by markodonnell-deriv — APPROVED** [review]
Looks good

**Pull request merged**